### PR TITLE
[Async functions] Make return values explaination more explicit

### PR DESCRIPTION
**Pull request by mendaomn**
The sentence wasn't really clear to me at first, and it took me a while to figure out the meaning of it. It may be just that English is not my native language, but I figured I'd suggest a more explicit version of the same sentence.

Hope this doesn't change the intended meaning and that it's written in correct English
Great article though,
Keep up!

### CODE REVIEW

**Comment by PaulKinlan** [comment]
Calling @jakearchibald

**Comment by petele** [comment]
@jakearchibald - will you take a quick peek and if it's good, approve then assign to me. if you want changes, use the review tool to request changes and assign back to @mendaomn.  Thanks!

File: src/content/en/fundamentals/getting-started/primers/async-functions.md [hunk 1]
```diff
@@ -65,8 +65,7 @@ easier to read, especially for those less familiar with promises.
 
 ## Async return values
 
-Calling an async function returns a promise for whatever the function returns or
-throws. So with:
+An async function's return value gets automatically wrapped in a Promise, thus, calling such async function returns a Promise for whatever the function returns or throws. So with:
```
Comment (petele) [inline src/content/en/fundamentals/getting-started/primers/async-functions.md]: Please wrap lines at no more than 80 characters.

**Comment by jakearchibald** [comment]
Grammar's a little off. I'll take a swing at it.

**Comment by petele** [comment]
Replaced with #3764

**Comment by petele** [comment]
@mendaomn - Thanks for the PR/issue!

**Comment by jakearchibald** [comment]
Also https://github.com/google/WebFundamentals/pull/3765. Thanks for pointing out the issue @mendaomn!

**Comment by mendaomn** [comment]
Thanks for taking a look at it!

**Pull request closed without merging**